sample-apps/react-native/dogfood/src/components/MeetingUI.tsx

```diff
@@ -55,12 +50,9 @@ export const MeetingUI = ({ callId, navigation, route }: Props) => {
     }
   }, [call, appStoreSetState]);
 
-  const onHangUpCallButtonHandler = async () => {
+  const onCallHangupHandler = async () => {
     setShow('loading');
     try {
-      if (callingState === CallingState.LEFT) {
-        return;
-      }
```

Comment:
Is this not needed anymore, what is the story behind it?

Reply:
Definitely needed. I removed it by mistake when doing the refactoring